packages/contracts-bridge/contracts/test/harness/IBridgeRouterHarness.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+// TODO: When we move away from Hardhat, we should change the relative path
```

Comment:
You could also add the entire test folder to the filter block in the hardhat config